README.md

```diff
@@ -153,7 +159,7 @@ have access to Dependabot alerts.)
 ## AWS Infrastructure deployment
 
 1. [Configure AWS](https://confluence.linz.govt.nz/display/GEOD/Login+to+AWS+Service+Accounts+via+Azure+in+Command+Line)
-1. Get AWS credentials (see: https://www.npmjs.com/package/aws-azure-login) for 12 hours:
+1. Get AWS credentials for 12 hours:
```

Comment:
Should this be `2.`?

Reply:
I prefer not doing that:

1. Markdown renderers convert it to a correctly numbered list, so it's not necessary.
1. Sequential numbering in the markdown leads to a bunch of unnecessary noise in diffs when changing lists.